includes/class-wcpdf-main.php

```diff
@@ -444,8 +444,15 @@ public function generate_document_ajax() {
 				}
 				break;
 			case 'full':
-				// no order
-				if ( ! $order ) {
+				// check the user privileges
+				$full_permission = WPO_WCPDF()->admin->user_can_manage_document( $document_type );
+
+				if ( $full_permission ) {
+                    break;
```

Comment:
I think you have a problem with indentation in your changes, must be tab with 4 spaces

Reply:
This is because of my IDE! I don't know what is wrong with it!
I hit `enter` to go to the next line, and sometimes it creates it with spaces instead of tabs! Or, I press the tab, and it replaces it with spaces! Weird!